osclib/stagingapi.py

```diff
@@ -1570,7 +1572,40 @@ def _candidate_adi_project(self):
             adi_index = i + 2
         return self.adi_prj_from_number(adi_index)
 
-    def create_adi_project(self, name):
+    def update_adi_frozenlinks(self, name, src_prj):
+        search = {
+            'package': "@project='%s' and devel/@project='%s'" % (self.project, src_prj),
+        }
+        try:
+            collection = osc.core.search(self.project, **search)['package']
+        except urllib2.HTTPError, e:
+            raise e
```

Comment:
Any reason to catch the exception if only raised when caught, otherwise drop try/except?